cdk/domino_cdk/aws_configurator.py

```diff
@@ -45,17 +53,30 @@ def install_calico(self):
             yaml = YAML(typ="safe")
             loaded_manifests = list(yaml.load_all(stream))
 
-            crds = eks.KubernetesManifest(
-                self.scope,
-                "calico-crds",
-                cluster=self.eks_cluster,
-                manifest=[crd for crd in loaded_manifests if crd["kind"] == "CustomResourceDefinition"],
+            crd_manifests += [crd for crd in loaded_manifests if crd["kind"] == "CustomResourceDefinition"]
+            notcrd_manifests += [notcrd for notcrd in loaded_manifests if notcrd["kind"] != "CustomResourceDefinition"]
+
+        # NB: be careful changing resource names or removing the manifests as prune is set by default
+        # and could inadvertantly remove other resources if deleted after the new manifest is created
+
+        # Split CRDs into multiple manifests so they don't go over the lambda limit of 262144 bytes
```

Comment:
Does this have any noticeable impact on install time?

Reply:
~30min for your last commit in master, ~31min for my last commit here